mssql/setup.sql

```diff
@@ -1,4 +1,7 @@
 CREATE DATABASE Test_OpenCorona;
+ALTER DATABASE Test_OpenCorona SET COMPATIBILITY_LEVEL = 100;
```

Comment:
:+1: TPP is actually using SQL Server 2016, but with the compatability level set to 100.  Which is annoying, because we'd get `string_split` with 2016, which would be nice

Reply:
That is annoying 🙁